environments/linux.container.lock

```diff
@@ -1,2 +1,2 @@
 // Generated File DO NOT MANUALLY EDIT, this file SHOULD be commited to version control system
-{"image_base_name":"tipibuild/tipi-ubuntu","manifest_digest":"sha256:b2a9606b584ec13dd22eb192e0d262a8b58b0e8d4035c8624cdddeceddc29aaf","platform":"linux/amd64","raw_envzip_hash":"25477f3bbbfc32659cc291b53074ebb5642c2177"}
\ No newline at end of file
+{"image_base_name":"tipibuild/tipi-ubuntu","manifest_digest":"sha256:ba36e028fad402037de9fee3bccc4033d04cc4494ac2138c03a8571e9b6b596c","platform":"linux/amd64","raw_envzip_hash":"f4faf71b1e4ee2361cc76d81b3737d9c87836b65"}
```

Comment:
It's weird that the raw_envzip_hash changed no ?

Reply:
`"image": "tipibuild/tipi-ubuntu:{{cmake_re_source_hash}}"` I think it's should change because cmake_re_source_hash changed between versions v0.0.74 and v0.0.75

How I managed to produce it :
- `export TIPI_INSTALL_VERSION=v0.0.75 && /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/tipi-build/cli/master/install/install_for_macos_linux.sh )"`
- `rm -rf ./environments/linux.container.lock`
- `cmake-re -S . -B ./build-hermetic -DCMAKE_TOOLCHAIN_FILE=environments/linux.cmake`